README.md

```diff
@@ -181,7 +184,7 @@ use Psr\Log\NullLogger;
 
 /** @var ConfiguredQuery $configuredQuery */
 
-$resultIterator = new ResultIterator(
+$configuredQuery = new ResultIterator(
```

Comment:
Don't see the point in this and the same change later on. Also the inline docs are outdated in such case.